pkg/viewer/transaction.go

```diff
@@ -115,3 +118,34 @@ func (t *TxHandler) Get(ta TransArg) (*models.Transaction, error) {
 	}
 	return tx, err
 }
+
+func (t *TxHandler) CountByOrg(ta TransArg) ([]string, []int64, error) {
+	if ta.NetworkName == "" {
+		return nil, nil, fmt.Errorf("network name can't be empty")
+	}
+
+	var creators []string
+	var counts []int64
+	txs := make([]models.Transaction, 0)
+	q := t.db.Model(&txs)
+
+	// query all creators
+	_, err := t.db.Query(&creators, `select distinct "creator" from transactions`)
+	if err != nil {
```

Comment:
```sql
select "creator", count(*) from transactions group by "creator"
```
It will group the data according to creator and count the number of data in each group.